test/maconn.browser.spec.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+
+import { multiaddr } from '@multiformats/multiaddr'
+import { expect } from 'chai'
+import { WebRTCMultiaddrConnection } from './../src/maconn'
+
+describe('Multiaddr Connection', () => {
+  it('can open and close', async () => {
```

Comment:
imho it doesn't really test that right? I mean it proves the `close` property changes when you call `close()` but it might as well be done using a dummy property which toggles.

Not better to next to what you already do, also write to it, and have it succeed proving you receive the value, with then the the write or so afterwards doesn't work and you instead get some specific error that indicates the conn is closed and that you also confirm the content is indeed not written.

And same for or instead of the reading part?